### PR TITLE
performance and accessibility (issue #5)

### PR DESCRIPTION
**Pull request by aafrida**

### CODE REVIEW

**Review by n-aggarwal — CHANGES_REQUESTED** [review]
If you look at the page speed insights for index.html you will see that the website is slow because of the embedded youtube videos. The image formats were but a minor issue, if you want to fix the performance issue, replace the youtube video with a lightweight static html webpage, perhaps one of our making.

**Comment by n-aggarwal** [comment]
Also since you changes from `public` to `assets` which i am not sure why in the first place, please make sure to update the readme file so that the logo works properly.

File: index.html [hunk 1]
```diff
@@ -76,17 +76,16 @@ <h1 class="heading">
         </div>
 
         <h1 class="heading">
-            Highly Rated Song
+            Mini Preview
         </h1>
         <div class="block">
-            <iframe width="420" height="345" src="https://www.youtube.com/embed/tgbNymZ7vqY">
-            </iframe>
+            <iframe loading="lazy" title="link"  width="420" height="345" src="index.html"></iframe>
```
Comment (ananafrida) [inline index.html:82]: replaced YT video with web link

**Comment by ananafrida** [comment]
YT video was replaced with a web link before the comment was made. For a specific reference, a comment is left in the code under specific changes. Please check it. 

The folder was renamed to follow the conventional naming (minor issue). Usually, images and other project assets remain under "asset" folder.

readme file is updated accordingly. For future reference, try not to put any tags other than <li> under <ul>. That causes accessibility decrement.

Performance and accessibility tested and requirements met. Please find the pictures below.
image
image

**Review by n-aggarwal — APPROVED** [review]

**Pull request merged**